tools/rcverify.sh

```diff
@@ -69,7 +72,7 @@ KEYS=$RC-$V-KEYS
 
 NOTICE=$(cat << END
 Apache $DESCRIPTION
-Copyright 2016-2021 The Apache Software Foundation
+Copyright $COPYRIGHT The Apache Software Foundation
```

Comment:
This needs to be configurable as IDE plugins use `2020-2021` in the NOTICE.txt.
https://github.com/apache/openwhisk-intellij-plugin/blob/master/NOTICE.txt#L2
https://github.com/apache/openwhisk-vscode-extension/blob/master/NOTICE.txt#L2